metrics/cli/test/cli/updater/factory/updater_factory_test.dart

```diff
@@ -0,0 +1,84 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:cli/cli/updater/factory/updater_factory.dart';
+import 'package:cli/cli/updater/updater.dart';
+import 'package:cli/common/model/services/services.dart';
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+
+import '../../../test_utils/matchers.dart';
+import '../../../test_utils/mock/firebase_service_mock.dart';
+import '../../../test_utils/mock/flutter_service_mock.dart';
+import '../../../test_utils/mock/gcloud_service_mock.dart';
+import '../../../test_utils/mock/git_service_mock.dart';
+import '../../../test_utils/mock/npm_service_mock.dart';
+import '../../../test_utils/mock/sentry_service_mock.dart';
+import '../../../test_utils/mock/services_factory_mock.dart';
+
+void main() {
+  group("UpdaterFactory", () {
+    final servicesFactory = ServicesFactoryMock();
+    final updaterFactory = UpdaterFactory(servicesFactory);
+    final gcloudService = GCloudServiceMock();
+    final flutterService = FlutterServiceMock();
+    final gitService = GitServiceMock();
+    final npmService = NpmServiceMock();
+    final firebaseService = FirebaseServiceMock();
+    final sentryService = SentryServiceMock();
```

Comment:
✅ 
It seems like we create a lot of mocks here that we don't even use as mocks. Maybe, for this purpose, we can introduce the stub Services class that contains default services to use so we can avoid unnecessary mocks in such cases?